src/pages/paper-wallet/paper-wallet.ts

```diff
@@ -0,0 +1,204 @@
+import { Component } from '@angular/core';
+import { ViewController, NavController, NavParams, ActionSheetController } from 'ionic-angular';
+import { Logger } from '@nsalaun/ng-logger';
+
+//providers
+import { BwcProvider } from '../../providers/bwc/bwc';
+import { OnGoingProcessProvider } from '../../providers/on-going-process/on-going-process';
+import { PopupProvider } from '../../providers/popup/popup';
+import { WalletProvider } from '../../providers/wallet/wallet';
+import { FeeProvider } from '../../providers/fee/fee';
+import { ProfileProvider } from '../../providers/profile/profile';
+
+//pages
+import { ScanPage } from '../scan/scan';
+
+import * as _ from 'lodash';
+
+@Component({
+  selector: 'page-paper-wallet',
+  templateUrl: 'paper-wallet.html',
+})
+export class PaperWalletPage {
+
+  public wallet: any;
```

Comment:
Use the `wallet` object params instead of assign all of this variables separately.